config/routes.rb

```diff
@@ -73,6 +73,10 @@
         post :remove
       end
     end
+
+    resource :partnership_csv_uploads, path: "partnership-csv-uploads", only: %i[new create] do
+      get "error-page", action: :error_page
+    end
```

Comment:
I think this should be inside `report_schools`, no? Like `confirm_schools` is

Reply:
That's probably right, I started on it before Alfred had finished, will look at adapting